server/app.js

```diff
@@ -41,4 +43,8 @@ app.use(function(err, req, res, next) {
   res.render('error');
 });
 
+const server = app.listen(3000, () => {
```

Comment:
Let's take the port number from env so it can be changed on different environments.

```suggestion
const server = app.listen(process.env.PORT || 3000, () => {
```

Reply:
i'm not sure what you mean by this. I see what you would like me to do but i'm not understanding why.